tests/unit/test_next_command.py

```
@@ -1,213 +1,737 @@
+        "utils_ROOT": mock_root_util,
+    }
+
+    patch.stopall()  # Stop all patches started with start()
+
+
+# Test Scenarios for run_next()
+
+
+@pytest.mark.parametrize(
+    "mock_config",
+    [pytest.lazy_fixture("mock_ai_provider_openai_direct")],
```

Pylint (3.3.7)

[error] Module 'pytest' has no 'lazy_fixture' member

(E1101)